### PR TITLE
Polishing Up "TestDox.php' PHPDoc Blocks for Clarity

### PR DESCRIPTION
**Pull request by Muhammad-Sarfaraz**
### What:

- [ ] Improvment of 'TestDox.php' PHPDoc Blocks

### Description:

Added the missing parenthesis and period for proper punctuation and formatted the doc block to meet PHPDocumentor standards.

### CODE REVIEW

**Comment by Muhammad-Sarfaraz** [comment]
This is my first PR to BestPHP oops, PestPHP! 😄 Please feel free to let me know if I've made any mistakes.

File: src/Factories/Annotations/TestDox.php [hunk 1]
```diff
@@ -14,13 +14,12 @@ final class TestDox implements AddsAnnotations
...
     public function __invoke(TestCaseMethodFactory $method, array $annotations): array
     {
-        /*
-       * escapes docblock according to
+      /*
+       * Escapes docblock according to
        * https://manual.phpdoc.org/HTMLframesConverter/default/phpDocumentor/tutorial_phpDocumentor.howto.pkg.html#basics.desc
        *
-       * note: '@' escaping is not needed as it cannot be the first character of the line (it always starts with @testdox
+       * note: '@' escaping is not needed as it cannot be the first character of the line (it always starts with @testdox).
```
Comment (owenvoke) [inline src/Factories/Annotations/TestDox.php]: Perhaps we can upper-case `Note` here too. 🤷🏻
```suggestion
       * Note: '@' escaping is not needed as it cannot be the first character of the line (it always starts with @testdox).
```

**Review by owenvoke — APPROVED** [review]
Hi @Muhammad-Sarfaraz, nice work. Thanks!

Just one suggestion, but otherwise looks great.

**Comment by Muhammad-Sarfaraz** [comment]
> Hi @Muhammad-Sarfaraz, nice work. Thanks!
> 
> Just one suggestion, but otherwise looks great.


Oops, I totally missed that! My bad, and thanks for catching it!

**Pull request merged**